### PR TITLE
Testsuite tweaks to ease Gentoo packaging integration.

### PR DESCRIPTION
**Pull request by trofi**

### CODE REVIEW

File: unifdefall.sh [hunk 1]
```diff
@@ -48,7 +48,7 @@ export LC_ALL=C
 # list of all controlling macros; assume these are undefined
 "$unifdef" $debug -s "$@" | sort -u | sed 's/^/#undef /' >"$tmp/undefs"
 # list of all macro definitions
-cc -E -dM "$@" | sort >"$tmp/defs"
+${CC-cc} -E -dM "$@" | sort >"$tmp/defs"
```
Comment (domo141) [inline unifdefall.sh]: I suggest ${CC:-cc} -- so that is CC is empty (but set) then cc is used, not ''

File: tests/multi-generic-sh [hunk 2]
```diff
@@ -9,7 +9,7 @@ unifdef -M~ $opts $mfiles
 e=$?
 for f in $mfiles
 do
-	diff -u $f~ $f | sed '/^[+-]\{3\} /s/	.*//'
+	LANG=C diff -u $f~ $f | sed '/^[+-]\{3\} /s/	.*//'
```
Comment (domo141) [inline tests/multi-generic-sh]: Good stuff, I had to do similar change to make this work for my fi_FI locale.

I would suggest to have export LC_ALL=C LANG=C just after export PATH line (line 3 in this scripts/runtests.sh)

File: unifdefall.sh (hunk 1 above)
Comment (trofi) [reply unifdefall.sh]: Sounds good! Updated.

File: tests/multi-generic-sh (hunk 2 above)
Comment (trofi) [reply tests/multi-generic-sh]: Sounds good! Updated.

**Comment by fanf2** [comment]
Hello!

Thanks for your contributions. I expect I will have time to look at this PR and the others in January.

**Pull request closed without merging**